def foo():
    # ERROR:
    foo(1, 2, 3, 4, 5)
    # ERROR:
    foo(5)
